README.md

```diff
@@ -180,6 +182,29 @@ NSX Management Proxy Sample `values.yaml`
 
 - Download latest version: [values for v0.1.1](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=supervisor-services&path=nsx-management-proxy/v0.1.1/nsx-management-proxy-data-values.yml). Make sure to fill the property `nsxManagers` with your NSX Manager IP(s).
 
+
+## Consumption Interface
+
+Provides the Local Consumption Interface (LCI) for Namespaces within vSphere Client. This also includes the Single Sign On (SSO) component required by the Cloud Consumption Interface (CCI) in Aria Automation within VMware Cloud Foundation.
+
+### Consumption Interface Versions
+
+Installation instructions can be found here: https://docs.vmware.com/en/VMware-Aria-Automation/8.17/Using-Automation-Service-Broker/GUID-C0D4DDE3-5592-46E2-8C79-381F7EFDD3D7.html
+
+Do not use the service definition YAML file linked in those instructions. Substitute the definition YAML file for the version you are installing from the list below.
```

Comment:
Is there a plan to have the instructions updated so they either include the right link (or no link)?

Reply:
Updated to point to generic instructions which have the user reference the YAMl they are downloading.